library/photon

```diff
@@ -4,20 +4,20 @@ Maintainers: Fabio Rapposelli <fabio@vmware.com> (@frapposelli),
 GitRepo: https://github.com/vmware/photon-docker-image.git
 Directory: docker
 
-Tags: 4.0, 4.0-20221017, latest
+Tags: 4.0, 4.0-20221021, latest
 Architectures: amd64, arm64v8
-GitFetch: refs/heads/x86_64/4.0-20221017
-GitCommit: 000b077bff7fbc19384dd9dbd7302396ba640977
-arm64v8-GitFetch: refs/heads/aarch64/4.0-20221017
-arm64v8-GitCommit: 488bb7a23caffe500b2b0cdc3171239ca5913720
+GitFetch: refs/heads/x86_64/4.0-20221021
+GitCommit: ad3ddb698615403aee953dbdfd555c4f23c8c77e
+arm64v8-GitFetch: refs/heads/aarch64/4.0-20221021
+arm64v8-GitCommit: 066d494852d91b5eb43371c226afb285d1076ac6
 
-Tags: 3.0, 3.0-20221017
+Tags: 3.0, 3.0-20221021
 Architectures: amd64, arm64v8
-GitFetch: refs/heads/x86_64/3.0-20221017
-GitCommit: 6dd5ddd31d79104fe1b03d28fc6738f5d7b3f094
-arm64v8-GitFetch: refs/heads/aarch64/3.0-20221017
-arm64v8-GitCommit: 7617dde2ef819109e96db048f41fdb306265cd36
+GitFetch: refs/heads/x86_64/3.0-20221021
+GitCommit: 80e57db4e9f064bb1efa7be8c3ec5ac85558766a
+arm64v8-GitFetch: refs/heads/aarch64/3.0-20221021
+arm64v8-GitCommit: af057be65f4cfd38584867fe15b02b9fff9b3fe7
 
-Tags: 2.0, 2.0-20220923
-GitFetch: refs/heads/2.0-20220923
-GitCommit: c65f7455abcf2c665afe6292fdd1cda48464c493
+Tags: 2.0, 
+GitFetch: refs/heads/
+GitCommit: 
```

Comment:
CC @YustasSwamp @frapposelli: It seems that your automation keeps trying to remove part of the `2.0` bits (and so this PR fails to attempt the tests and verification).